src/schema/document.rs

```diff
@@ -97,6 +98,11 @@ impl Document {
         self.add_field_value(field, value);
     }
 
+    /// Add a IP address field
+    pub fn add_ip_addr(&mut self, field: Field, value: IpAddr) {
```

Comment:
```suggestion
    pub fn add_ip_addr(&mut self, field: Field, value: IpAddrV6) {
```

Reply:
That's a tricky one. We probably want to drop the notion of IpAddr to only work with IpAddrV6, and document everywhere if we get an ipv4 when say, we parse json, it gets mapped ot IpAddrV6.

In other word, "code" gets and returns  `std::net::IpAddrV6`.
"data" when interpreted, can get IPv4 representations, but converts them to IPv6. Does that make sense?